runway/module/cloudformation.py

```diff
@@ -106,57 +106,56 @@ def run_stacker(self, command='diff'):  # pylint: disable=too-many-branches
         if stacker_env_file_present:
             stacker_cmd.append(stacker_env_file)
 
-        with change_dir(self.path):
-            # Iterate through any stacker yaml configs to deploy them in order
-            # or destroy them in reverse order
-            for _root, _dirs, files in os.walk(self.path):
-                for name in (
-                        reversed(sorted(files))
-                        if command == 'destroy'
-                        else sorted(files)):
-                    if re.match(r"runway(\..*)?\.yml", name) or (
-                            name.startswith('.')):
-                        # Hidden files (e.g. .gitlab-ci.yml) or runway configs
-                        # definitely aren't stacker config files
-                        continue
-                    if os.path.splitext(name)[1] in ['.yaml', '.yml']:
-                        if not (stacker_env_file_present or self.options.get(
```

Comment:
I can't find a reason why this would fail for a specific Stacker file, and not for all of them.  So I moved it outside the `with`